examples/rpi-pico-w/src/main.rs

```diff
@@ -227,4 +227,10 @@ impl cyw43::SpiBusCyw43 for MySpi {
         self.read(read).await;
         self.cs.set_high();
     }
+
+    async fn wait_for_event(&mut self) {}
+
+    fn clear_event(&mut self) {}
```

Comment:
Is it possible to remove this, and make `wait_for_event` clear the flag? It feels somewhat "implementation detail" to me.